style.css

```
@@ -259,3 +259,87 @@ img {
   padding-top: 10px;
 }
 
+/*Qualidades*/
+
+.qualidades_container {
+	max-width: 1150px;
+	padding: 60px 0 0 15px;
```

Aqui acho que a gente pode deixar assim.
```suggestion
	padding: 60px 15px;
```